draft-ietf-quic-recovery.md

```diff
@@ -842,6 +842,10 @@ congestion, as TCP does with a Retransmission Timeout (RTO) {{?RFC5681}}.
 The RECOMMENDED value for kPersistentCongestionThreshold is 3, which is
 approximately equivalent to two TLPs before an RTO in TCP.
 
+The persistent congestion period SHOULD NOT start until there is at
+least one RTT sample, both because the length of the period is unknown and
+the PTO may be excessively conservative.
```

Comment:
Maybe a wordier version is clearer:

"Persistent congestion SHOULD NOT consider packets that were sent prior to obtaining an RTT sample.  Correctly detecting persistent congestion can depend on packets being sent on PTO expiration.  The initial RTT, which is used to set the PTO timer, might be too conservative.  If the first RTT sample results in a much smaller RTT, the resulting persistent congestion interval might contain too few packets for their loss to be indicative of congestion."

Reply:
I find the first sentence potentially confusing, but I can make this more verbose along the lines of the second two sentences.